src/recipe.js

```diff
@@ -7,35 +7,30 @@ class Recipe {
     this.name = name;
     this.tags = tags;
   }
-  returnIngredientNames(ingredientData) {
+  findIngredients(ingredientData) {
     const recipeIngredients = [];
     this.ingredients.forEach((recipeIngredient) => {
```

Comment:
remove parenthesis around recipeIngredient

Reply:
Good catch. Fixed it and pushed it up!